frontend/src/shared/components/Modal.tsx

```diff
@@ -14,9 +14,9 @@ PURPOSE.  See the GNU General Public License for more details.
 You should have received a copy of the GNU General Public License along with this program.
 If not, see <https://www.gnu.org/licenses/>.
 */
-import React from 'react';
-import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
-import MaterialModal from '@material-ui/core/Modal';
+import React from "react";
```

Comment:
Please revert all changes to this file. I have made edits in a separate ticket to address the responsive issues :)